traceview/discovery.py

```diff
@@ -17,9 +17,7 @@ class Action(Resource):
     PATH = "actions"
 
 
-class App(Resource):
-
-    PATH = "apps"
+# for App class, see app.py
```

Comment:
@dankosaur so i guess this comment was added because of the new "App Configuration" section in the documentation, which moves "App" out of "discovery":

http://dev.appneta.com/docs/api-v2/app.html

Is this assumption correct? Also, when will the docs be updated for the new delete functionality?